frontends/web/src/components/TaskOwnerPageComponents/Settings.js

```diff
@@ -348,6 +349,49 @@ const Settings = (props) => {
                           />
                         </Col>
                       </Form.Group>
+                      <Form.Group
+                        as={Row}
+                        controlId="context"
+                        className="py-3 my-0 border-bottom"
+                      >
+                        <Form.Label column>
+                          Context strategy
```

Comment:
Maybe we should change this from "Context strategy" to "Sampling of Contexts In the Create Interface" so users know this is about the create interface?